package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "bundler",
   "version": "1.0.9",
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
I got tired of seeing the warning message every time I ran an npm command. Just adding this in so it'll go away.